msteams-platform/graph-api/rsc/resource-specific-consent.md

```diff
@@ -374,54 +307,279 @@ Example for RSC in a chat
 ```
 
 > [!NOTE]
-> If the app is meant to support installation in both team and chat scopes, then both team and chat permissions can be specified in the same manifest under `applicationPermissions`.
+> If an app is meant to support installation in both team and chat scopes, then both team and chat permissions can be specified in the same manifest under `applicationPermissions`.
 
 <br>
+</details>
+
+Whenever an app is installed by an authorized user within Teams, the RSC permissions requested in the app’s manifest are shown to the user and consequently granted as part of the app installation process.
+
+## Grant RSC permissions to an app
+
+Users' ability to grant RSC permissions varies based on resource types and access modes. The following are the types of RSC permissions for an app:
+
+* **Application context RSC permissions**: Users authorized to install an app in a specific scope have the right to grant any RSC permissions requested by the app in that specific scope. In addition to installing apps in a specific scope, the following constraints apply for the user to grant RSC permission to access data in app-only mode:
```

Comment:
"Users authorized to install an app in a specific scope have the right to grant any RSC permissions requested by the app in that specific scope."
This statement is confusing. 
Why not just say that the user should be able to install the app and be one of the following: (other constaints)

Reply:
This content already incorporated in the new structured doc PR #8221